examples/fmriprep-singularity/README.md

```diff
@@ -66,7 +66,8 @@ Configure the dataset in which you want to collect the results of the (re)comput
 Add a `datalad-remake` special remote:
 
 ```bash
-> git annex initremote datalad-remake type=external externaltype=datalad-remake encryption=none allow-untrusted-execution=true
+> git annex initremote datalad-remake type=external externaltype=datalad-remake \
+encryption=none allow-untrusted-execution=true autoenable=true
 ```
```

Comment:
This is no longer required (see above)

Reply:
Done in [f45274a](https://github.com/datalad/datalad-remake/pull/62/commits/f45274a254ba9d2c47f9a45ba37df344b6382687).